.github/workflows/ossf-scorecard-reporting.yml

```diff
@@ -17,15 +18,19 @@ jobs:
   security-scoring:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
       - name: OpenSSF Scorecard Monitor
-        uses: UlisesGascon/openssf-scorecard-monitor@v1.0.1
+        uses: UlisesGascon/openssf-scorecard-monitor@b2fa3a9589511a7d60da9e3250617946a6c0360d # v2.0.0-beta1
         with:
           scope: tools/ossf_scorecard/scope.json
           database: tools/ossf_scorecard/database.json
```

Comment:
Are these configurations still needed? I was wondering since you deleted the files.

Reply:
I can keep the database one as it is if we want to keep the last records, the scope structure [has change a bit](https://github.com/UlisesGascon/openssf-scorecard-monitor/blob/main/schemas/scope.json) so I preferred to removed ir especially if we want to do a discovery mode (so the org repos will be auto-included)

Reply:
Yes I come to understand for database.